Documentation/gettingstarted/clustermesh/aks-clustermesh-prep.rst

```diff
@@ -0,0 +1,237 @@
+.. _gs_clustermesh_aks_prep:
+
+**********************************
+AKS-to-AKS clustermesh preparation
+**********************************
+
+This is a step-by-step guide on how to install and prepare 
+AKS (Azure Kubernetes Service) clusters in BYOCNI mode to meet the requirements 
+for the clustermesh feature.
+
+In this guide we will install two AKS clusters in BYOCNI(Bring Your Own CNI) 
```

Comment:
English prose uses a space before opening parenthesis.

`sed -i 's/\([^$ ]\)(/\1 (/g' Documentation/gettingstarted/clustermesh/aks-clustermesh-prep.rst`

Reply:
Ah, I was under the impression that the brackets should be adjacent without a space when explaining a acronym or abbreviation. Good to know! Thanks for the review.